cmd/juju/model/checkout.go

```diff
@@ -20,10 +20,9 @@ import (
 const (
 	checkoutSummary = "Work on the supplied branch."
 	checkoutDoc     = `
-Switch to the supplied branch, causing changes to charm configuration,
-charm URL or resources to apply only to units tracking the branch.
-Changing the branch to "master" causes changes to be applied to all units
-as usual.
+Switch to the supplied branch, causing changes to charm configuration to apply 
+only to units tracking the branch. Changing the branch to "master" causes 
+changes to be applied to all units as usual.
```

Comment:
I don't quite follow the last sentence, does this mean if you have changes on a unit and switching to "master" branch will cause them to be staged and committed?

Reply:
I've clarified to better represent master branch behaviour.